src/updater.cpp

```diff
@@ -151,10 +151,12 @@ void Updater::downloadUpdate(QString link)
     dialog.exec();
     process->start("bash",
                    { "-c",
-                     "cd $TMPDIR && rm -rf /Applications/minikube-gui.app/ && mv "
-                     "minikube-gui-macos/minikube-gui.app/ /Applications/ && rm -rf "
-                     "minikube-gui-macos && open -R /Applications/minikube-gui.app/ && open "
-                     "/Applications/minikube-gui.app/" });
+                     "cd $TMPDIR && rm -rf /Applications/minikube-gui.app/ &&"
+                     "mv minikube-gui-macos/bin/minikube-gui.app /Applications/ &&"
+                     "rm -rf minikube-gui-macos && "
+                     "xattr -d com.apple.quarantine /Applications/minikube-gui.app; true &&"
+                     "open -R /Applications/minikube-gui.app/ && "
```

Comment:
we can get rid of open -R that was meant to open the finder so the user manually opens the App
now that we xattr it we dont need to do that anymore !